pkg/stanza/docs/operators/headerless_jarray_parser.md

```diff
@@ -0,0 +1,145 @@
+## `headerless_jarray_parser` operator
+
+The `headerless_jarray_parser` operator parses the string-type field selected by `parse_from` with the given header values.
```

Comment:
Thanks for adding a readme too. I think it should clearly describe the expected format, including linking to a specification if possible. When I look for a format called "jarray", I find C# classes and such which are not clearly related. Is this link appropriate? https://json-schema.org/understanding-json-schema/reference/array

Reply:
Moreover, should the parser be called `json_array_parser`?

Reply:
yeah i know what you mean about googling it.. Looks like "json array" gives better results and chatgpt did a good job of explaining it to me hehe..

I updated the readme with examples for json arrays and what is expected and also added that link in there.

the renaming to `json_array_parser` is also done